docker compose environment variables encoded as KEY='value' are not longer accepted since 2.6

Gotify 2.6.1 is ignoring the postfix host settings. The same connection settings work fine in 2.5. After upgrading to 2.6.1, gotify only attempts to connect to [::1]:5432

I cannot test this on Gotify 2.6.0 due to bool value crashing.

Rolling back to 2.5 works. The configuration is never changed.

Digging through changes, I can't really find much except
[2d6601747b546c81d364a87bcfaf070bdd4f9f74](https://github.com/gotify/server/commit/2d6601747b546c81d364a87bcfaf070bdd4f9f74)

I don't know if this directly correlates.

**Can the issue be reproduced with the latest available release? (y/n)** yes

**Which one is the environment gotify server is running in?**
- Docker with compose

<details><summary>Docker startup command or config file here (please mask sensitive information)</summary><br>
compose configuration snippit<br>

gotify:                                                                                                               
    container_name: gotify
    image: gotify/server:latest
    #image: gotify/server:2.5 # working image
    restart: unless-stopped
    environment:
    - TZ=America/Los_Angeles
    - GOTIFY_SERVER_PORT=<port_number>
    - GOTIFY_SERVER_SSL_ENABLED=false
    - GOTIFY_SERVER_SSL_REDIRECTTOHTTPS=false
    - GOTIFY_DATABASE_DIALECT=postgres
    - GOTIFY_DATABASE_CONNECTION='host=<ip_of_postgres_server> port=<port_of_postgres_server> user=<username> dbname=<dbname> password=<password> sslmode=<sslmode>'
    - GOTIFY_PASSSTRENGTH=11
    - GOTIFY_REGISTRATION=false
    - GOTIFY_SERVER_KEEPALIVEPERIODSECONDS=75
    - GOTIFY_SERVER_STREAM_PINGPERIODSECONDS=30
    volumes:
    - <volume>

</details>

**Do you have an reverse proxy installed in front of gotify server? (Please select None if the problem can be reproduced without the presense of a reverse proxy)**
- I do have a reverse proxy, but I can replicate without it, so choosing None

**On which client do you experience problems? (Select as many as you can see)**
- Backend issue


**What did you do?**
Upgraded

**What did you expect to see?**
Gotify server running

**What did you see instead? (Include screenshots, android logcat/request dumps if possible)**

Starting Gotify version 2.6.1@2024-11-16-08:49:43                                                                                                                                                               
panic: dial tcp [::1]:5432: connect: connection refused                                                                                                                                                         
                                                                                                                                                                                                                
goroutine 1 [running]:                                                                                                                                                                                          
main.main()                                                                                                                                                                                                     
        /src/gotify/app.go:47 +0x39c                                                                                                                                                                            
Starting Gotify version 2.6.1@2024-11-16-08:49:43                                                                                                                                                               
panic: dial tcp [::1]:5432: connect: connection refused                                                                                                                                                         
                                                                                                                                                                                                                
goroutine 1 [running]:                                                                                                                                                                                          
main.main()                                                                                                                                                                                                     
        /src/gotify/app.go:47 +0x39c                                                                                                                                                                            
Starting Gotify version 2.6.1@2024-11-16-08:49:43                                                                                                                                                               
panic: dial tcp [::1]:5432: connect: connection refused                                                                                                                                                         
                                                                                                                                                                                                                
goroutine 1 [running]:                                                                                                                                                                                          
main.main()                                                                                                                                                                                                     
        /src/gotify/app.go:47 +0x39c                                                                                                                                                                            
Starting Gotify version 2.6.1@2024-11-16-08:49:43                                                                                                                                                               
panic: dial tcp [::1]:5432: connect: connection refused                                                                                                                                                         
                                                                                                                                                                                                                
goroutine 1 [running]:                                                                                                                                                                                          
main.main()                                                                                                                                                                                                     
        /src/gotify/app.go:47 +0x39c

--- a/docs/installation.md
+++ b/docs/installation.md
@@ -41,9 +41,12 @@ services:
     ports:
       - 8080:80
     environment:
-      - GOTIFY_DEFAULTUSER_PASS=custom
+      GOTIFY_DEFAULTUSER_PASS: 'my-com"plex.\password'
     volumes:
       - "./gotify_data:/app/data"
+    # to run gotify as a dedicated user:
+    # sudo chown -R 1234:1234 ./gotify_data
+    user: "1234:1234"
 ```
 
 </p>

--- a/docs/installation.md
+++ b/docs/installation.md
@@ -41,12 +41,12 @@ services:
     ports:
       - 8080:80
     environment:
-      GOTIFY_DEFAULTUSER_PASS: 'my-com"plex.\password'
+      GOTIFY_DEFAULTUSER_PASS: 'admin'
     volumes:
       - "./gotify_data:/app/data"
     # to run gotify as a dedicated user:
     # sudo chown -R 1234:1234 ./gotify_data
-    user: "1234:1234"
+    # user: "1234:1234"
 ```
 
 </p>

